src/yarle.ts

```diff
@@ -31,34 +31,42 @@ export const parseStream = async (options: YarleOptions): Promise<void> => {
   let noteNumber = 0;
   let failed = 0;
   let skipped = 0;
-  
+
   const notebookName = utils.getNotebookName(options.enexSource);
 
   return new Promise((resolve, reject) => {
-    
+
     const logAndReject = (error: Error) => {
       console.log(`Could not convert ${options.enexSource}:\n${error.message}`);
       ++failed;
 
       return reject();
     };
-    if (!fs.existsSync(options.enexSource))
+    if (!fs.existsSync(options.enexSource)) {
       return logAndReject({name: 'NoSuchFileOrDirectory', message: 'source Enex file does not exists'});
-    
+    }
+
     const xml = flow(stream);
```

Comment:
Btw, you can pass some options to `flow()`, but I found now option that prevented the collapse of the note-attributes tag. Except maybe preserveMarkup, but that makes everything complicated again.